src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,35 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector
```

Comment:
The `Injector` is initialized with the string `"mate.academy.dao"`. Ensure that this string matches the package structure and configuration used in your project for dependency injection. If the package name is incorrect, it could lead to runtime errors.